### PR TITLE
VW: Explicit signal pass-through

### PR DESCRIPTION
**Pull request by jyoung8607**
Split from/prep for #28020.

### CODE REVIEW

File: selfdrive/car/volkswagen/mqbcan.py [hunk 1]
```diff
@@ -23,11 +31,23 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
...
+    "GRA_Verstellung_Zeitluecke",  # ACC button, time gap adj
+    "GRA_Typ_Hauptschalter",       # ACC main button type
+    "GRA_Codierung",               # ACC button configuration/coding
+    "GRA_Tip_Stufe_2",             # unknown related to stalk type
+    "GRA_ButtonTypeInfo",          # unknown related to stalk type
+  ]}
 
   values.update({
-    "COUNTER": counter,
+    "COUNTER": (gra_stock_values["COUNTER"] + 1) % 16,
```
Comment (sshane) [inline selfdrive/car/volkswagen/mqbcan.py:50]: if we're just passing it through, it should be fine to either use the stock counter or use our own?

Comment (sshane) [inline selfdrive/car/volkswagen/mqbcan.py:50]: nvm i see this is buttons

**Comment by sshane** [comment]
ty!

File: selfdrive/car/volkswagen/pqcan.py [hunk 2]
```diff
@@ -23,11 +31,24 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
...
+    "GRA_Recall",           # ACC button, resume
+    "GRA_Zeitluecke",       # ACC button, time gap adj
+    "GRA_Sender",           # ACC button, CAN message originator
+  ]}
 
   values.update({
-    "COUNTER": counter,
+    "COUNTER": (gra_stock_values["COUNTER"] + 1) % 16,
     "GRA_Abbrechen": cancel,
     "GRA_Recall": resume,
```
Comment (sshane) [inline selfdrive/car/volkswagen/pqcan.py:53]: looks like you can remove these from above unless you want the comments

File: selfdrive/car/volkswagen/mqbcan.py [hunk 3]
```diff
@@ -23,11 +31,23 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
...
     "GRA_Abbrechen": cancel,
     "GRA_Tip_Wiederaufnahme": resume,
```
Comment (sshane) [inline selfdrive/car/volkswagen/mqbcan.py:52]: same

File: selfdrive/car/volkswagen/pqcan.py (hunk 2 above)
Comment (jyoung8607) [reply selfdrive/car/volkswagen/pqcan.py:53]: #28246

File: selfdrive/car/volkswagen/mqbcan.py (hunk 3 above)
Comment (jyoung8607) [reply selfdrive/car/volkswagen/mqbcan.py:52]: #28246

**Pull request merged**